main/res/drawable/ic_menu_pocket_query.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
According to my research that would mainly mean, that the map activities will have to use our old fake actionbar for all API versions below 23.